matrix-sdk-android/src/main/java/org/matrix/android/sdk/api/util/BuildVersionSdkIntProvider.kt

```diff
@@ -26,6 +28,7 @@ interface BuildVersionSdkIntProvider {
      * Checks the if the current OS version is equal or greater than [version].
      * @return A `non-null` result if true, `null` otherwise.
      */
+    @ChecksSdkIntAtLeast(parameter = 0, lambda = 1)
```

Comment:
TIL :100:

Reply:
This was also new to me, I learn about it looking at the lint warning. Very useful!